contrib/local-environment/Makefile

```diff
@@ -1,34 +1,42 @@
 .PHONY: up
 up:
-	docker-compose up -d
+	docker compose up -d
```

Comment:
This is part of the docker CLI now? TIL

Reply:
Since 2020
https://docs.docker.com/compose/migrate/

> Compose V2, which was first released in 2020, is included with all currently supported versions of Docker Desktop. It offers an improved CLI experience, improved build performance with BuildKit, and continued new-feature development.